app/views/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user, html: { class: "form-horizontal" } do |f|
```

Comment:
The issue with your form display is that the gem now uses `layout: :horizontal`. You can read about it in the gem docs.